report/html/template.go

```diff
@@ -92,7 +92,7 @@ const templateContent = `
           <div className="columns">
               <div className="column is-three-quarters">
                 <strong className="break-word">{ this.props.data.file } (line { this.props.data.line })</strong>
-                <p>{ this.props.data.details }</p>
+                <p>{this.props.data.rule_id} (CWE-{this.props.data.cwe.id}): { this.props.data.details }</p>
```

Comment:
This shall do the trick
```suggestion
                <p>{this.props.data.rule_id} ({this.props.data.cwe.SprintID()}): { this.props.data.details }</p>
```

Reply:
SprintID() is not a function of the JavaScript data object

Reply:
You're right ! I thought this was a template with direct access to go functions